### PR TITLE
Map some net.minecraft.data stuff

### PR DESCRIPTION
**Pull request by NoahvdAa**

### CODE REVIEW

File: mappings/net/minecraft/state/property/Property.mapping [hunk 1]
```diff
@@ -45,4 +45,5 @@ CLASS net/minecraft/unmapped/C_vzlztuyw net/minecraft/state/property/Property
 			ARG 2 value
 		METHOD equals (Ljava/lang/Object;)Z
 			ARG 1 o
-		METHOD m_sxyiqvmx getValue ()Ljava/lang/Comparable;
+		METHOD m_sxyiqvmx value ()Ljava/lang/Comparable;
```
Comment (NoahvdAa) [inline mappings/net/minecraft/state/property/Property.mapping:48]: This class is a record, so this getter wasn't mapped properly

File: mappings/net/minecraft/data/server/RecipesProvider.mapping [hunk 2]
```diff
@@ -276,6 +276,8 @@ CLASS net/minecraft/unmapped/C_hwlnmobw net/minecraft/data/server/RecipesProvide
 		ARG 0 cache
 		ARG 1 json
 		ARG 2 path
+	METHOD m_uvcwjlge (Ljava/util/Set;Lnet/minecraft/unmapped/C_sdjwtmhs;Ljava/nio/file/Path;Lnet/minecraft/unmapped/C_qoyuyccf;)V
+		ARG 3 recipe
```
Comment (NoComment1105) [inline mappings/net/minecraft/data/server/RecipesProvider.mapping:280]: Are there no other arguments to this method?

File: mappings/net/minecraft/data/client/model/Model.mapping [hunk 3]
```diff
@@ -26,3 +28,6 @@ CLASS net/minecraft/unmapped/C_zppcsbsk net/minecraft/data/client/model/Model
 		ARG 1 block
 		ARG 2 texture
 		ARG 3 modelCollector
+	METHOD m_yevvcamp (Lcom/google/gson/JsonObject;Lnet/minecraft/unmapped/C_pixosdqo;Lnet/minecraft/unmapped/C_ncpywfca;)V
+		ARG 1 key
+		ARG 2 identifier
```
Comment (triphora) [inline mappings/net/minecraft/data/client/model/Model.mapping]:
```suggestion
		ARG 2 id
```

**Review by triphora — APPROVED** [review]

File: mappings/net/minecraft/data/server/RecipesProvider.mapping (hunk 2 above)
Comment (NoahvdAa) [reply mappings/net/minecraft/data/server/RecipesProvider.mapping:280]: Nope, it's a lambda passed to `RecipesProvider#generate`

**Review by OroArmor — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by IotaBread — APPROVED** [review]

**Pull request merged**